plugin/src/main/java/org/openrewrite/gradle/RewritePlugin.java

```diff
@@ -70,8 +71,38 @@ public void apply(Project project) {
 
         // Rewrite module dependencies put here will be available to all rewrite tasks
         Configuration rewriteConf = project.getConfigurations().maybeCreate("rewrite");
+        rewriteConf.withDependencies(deps -> {
```

Comment:
According to the documentation, a plugin author should not mutate the dependencies of said configuration within this method.

https://docs.gradle.org/current/javadoc/org/gradle/api/artifacts/Configuration.html#withDependencies(org.gradle.api.Action)

I think using `conf.getDependencies().addLater(..)` should be workable with the Gradle 4.10 floor from a supported tooling standpoint.

Reply:
Easily done. I'll push an updated version shortly.

Reply:
Done. I also fixed the `dependencyInScript` test that had an incorrect `version` parameter. Jackson has a 2.18.1 version that was causing it to fail. Not sure about the lombok test, other than it appears to be failing on the mainline as well.